app/assess/routes.py

```diff
@@ -467,3 +463,84 @@ def continue_assessment(application_id):
         section="NA",
         page_to_render="continue_assessment.html",
     )
+
+
+@assess_bp.route("/application/<application_id>/export", methods=["GET"])
+@login_required(roles_required=["LEAD_ASSESSOR"])
+def generate_doc_list_for_download(application_id):
+    current_app.logger.info(
+        f"Generating docs for application id {application_id}"
+    )
+    state = get_banner_state(application_id)
+    short_id = state.short_id[-6:]
+    latest_flag = get_latest_flag(application_id)
+    if latest_flag:
+        determine_display_status(state, latest_flag)
```

Comment:
this will need refactoring now as per https://github.com/communitiesuk/funding-service-design-assessment/pull/182 if you are actually using these properties?

Reply:
have removed this now as we're not using anymore 👍🏾